internal/provider/incident_catalog_entries_resource.go

```diff
@@ -600,3 +622,58 @@ func (r *IncidentCatalogEntriesResource) reconcile(ctx context.Context, data *In
 
 	return catalogType, entries, nil
 }
+
+// isAttributeManaged checks if the given attribute should be managed by this resource.
+func (m *IncidentCatalogEntriesResourceModel) isAttributeManaged(attributeID string) bool {
+	// Check if the attribute is in the managed list
+	attrSet, known := m.managedAttributesSet()
+	if !known {
+		return true
```

Comment:
I think this comment is wrong - what this checks is that you have the list right. So if you don't have the list, all attributes are managed. Caught me out so I would update!